packages/ui/src/lib/dropdown/Dropdown.svelte

```diff
@@ -0,0 +1,210 @@
+<script lang="ts">
+import { faCaretDown, faCheck } from '@fortawesome/free-solid-svg-icons';
+import Fa from 'svelte-fa';
+
+export let id: string | undefined = undefined;
+export let name: string | undefined = undefined;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export let value: any = '';
+export let disabled: boolean = false;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export let onInput: (val: any) => void = () => {};
+
+interface Option {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I don't know why we have so many 'any' there
it looks like we're having more than 10 // eslint-disable-next-line @typescript-eslint/no-explicit-any


should we have a generics for the type of the field we want to handle ?
or is it just `unknown`

Reply:
Most should be unknown, updated.